recipes/lrez/meta.yaml

```diff
@@ -1,36 +1,45 @@
-{% set name = "lrez" %}
-{% set version = "1.1" %}
-{% set sha256 = "5daac98361ff45f454e1d24ff33f259958d4c94bb627ee01a4beacd5b959db28" %}
+{% set version = "2.0" %}
+{% set sha256 = "9223e915644915b80d3adbe5f2693d9fde80bda1df6f2a2fd773c134c6e73356" %}
 
 package:
   name: lrez
-  version: {{ version }} 
-
-build:
-  number: 1
+  version: '{{version}}'
 
 source:
-  url: https://github.com/flegeai/LRez/archive/{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: "https://github.com/morispi/LRez/releases/download/v{{version}}/LRez.tar.gz"
+  sha256: '{{sha256}}'
+
+build:
+  skip: True  # [osx]
+  number: 0
 
 requirements:
   build:
     - make
-    - {{ compiler('cxx') }}
+    - {{ compiler('c')}}
+    - {{ compiler('cxx')}}
     - cmake
+    - wget
   host:
-    - htslib
+    - zlib
+    - boost-cpp
   run:
-    - htslib
-    - python>=3
-    - indexed_gzip
-    
+    - zlib
```

Comment:
```suggestion
```
`zlib` has "`run_exports`" entries which automatically add `zlib` to `run` when it's in `host`.